includes/render_config.h

```diff
@@ -70,17 +70,18 @@
 //Legacy - not needed on new installs
 //#undef METATILEFALLBACK
 
-// Metatiles are much larger in size so we don't need big queues to handle large areas
+// Typical osm.org render server can render 7 metatiles per second, and has average load of 4.5 metatiles per second.
+// Queue depth of (7 - 4.5) * 86400 = 216000 moves the load from the busy part of the day to the night time
 #ifdef METATILE
 #define QUEUE_MAX (64)
 #define REQ_LIMIT (32)
-#define DIRTY_LIMIT (1000)
-
+#define DIRTY_LIMIT (216000)
+#define HASHIDX_SIZE (477856)
```

Comment:
According to @gardster, this is not required and is redefined anyway later. I did it by analogy to the below. It looks like it's safe to remove both.